### ISSUE TITLE
update permission_handler 

### ISSUE THREAD

**Issue by Joseph-Nathan** [flutter-moum/flutter_screenshot_callback#48]
Because screenshot_callback >=1.1.3 depends on permission_handler ^5.0.1+1 and [MyApp]  depends on permission_handler ^8.0.0+1, screenshot_callback >=1.1.3 is forbidden.

**Comment by dragonkhoi** [comment]
You can use this branch while you wait! Just add to your pubspec.yaml. Thanks @devplanet-dp!
```
dependency_overrides:
  screenshot_callback: 
    git: 
      url: https://github.com/devplanet-dp/flutter_screenshot_callback.git
```

**Comment by ousvat** [comment]
fixed here: #50

**Comment by ousvat** [comment]
You can also use this package from my repo where I did more fixes:

```
screenshot_callback:
    git:
      url: git://github.com/ousvat/flutter_screenshot_callback.git
 ```
